### PR TITLE
Revert "Disable codeql analysis (for now?), it keeps hanging."

### PR DESCRIPTION
**Pull request by measure-fi**
This reverts commit b357deaebdf21fa5e8b587db28171d6c98526a22.

Testing codeql analysis on master.

### CODE REVIEW

**Comment by taariq** [comment]
Did we fix why CodeQL was hanging?

**Comment by taariq** [comment]
@measure-fi closing this on master. Will you create a CodeQL branch? We can always test on that branch and sync with master as needed.

File: .github/workflows/codeql-analyzer.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: "Code Scanning - Action"
+
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    branches: [ main ]
```
Comment (measure-fi) [inline .github/workflows/codeql-analyzer.yml:8]: Huh, this should be master.

**Comment by taariq** [comment]
> @taariq These results are most easily viewed as an open pull request, we may or may not decide to merge at this time. At present I would just like to be able to view the results.

Let's keep it rocking. Let's make sure this doesn't hang (it looks like it didn't) and merge to master after your review. My interest is just to avoid hanging waiting on this.

**Pull request closed without merging**